test/sym/unop.t

```diff
@@ -40,109 +56,172 @@ unop i64:
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 42)))
   Trap: unreachable
   Model:
     (model
-      (symbol_0 i64 (i64 1099511627776))
-      (symbol_1 f32 (f32 -36893492545465614336.000000)))
+      (symbol_0 i64 (i64 1))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 4242)))
   Trap: unreachable
   Model:
     (model
-      (symbol_0 i64 (i64 4194304))
-      (symbol_1 f32 (f32 -36893492545465614336.000000)))
+      (symbol_0 i64 (i64 1))
+      (symbol_2 f32 (f32 0.000000))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 -2147483648)))
   Trap: unreachable
   Model:
     (model
-      (symbol_0 i64 (i64 65536))
-      (symbol_1 f32 (f32 -36893492545465614336.000000)))
-  Reached 6 problems!
+      (symbol_0 i64 (i64 1))
+      (symbol_2 f32 (f32 inf))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 -2147483648)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_0 i64 (i64 131072))
+      (symbol_2 f32 (f32 -inf))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 0)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_0 i64 (i64 32768))
+      (symbol_2 f32 (f32 -inf))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 -2147483648)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_0 i64 (i64 4294967296))
+      (symbol_2 f32 (f32 -inf))
+      (symbol_3 f64 (f64 0.000000))
+      (symbol_1 i32 (i32 0)))
+  Reached 9 problems!
 unop f32:
   $ dune exec owi -- sym unop_f32.wast --no-stop-at-failure
   Trap: unreachable
   Model:
     (model
-      (symbol_0 f32 (f32 42.000000)))
+      (symbol_2 i32 (i32 1109917696)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_2 i32 (i32 -2147483647))
+      (symbol_0 f32 (f32 122.000977)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_2 i32 (i32 -2147479552))
+      (symbol_0 f32 (f32 123.007812)))
+  Trap: unreachable
+  Model:
+    (model
+      (symbol_2 i32 (i32 -2147479552))
+      (symbol_0 f32 (f32 115.007812)))
   Trap: unreachable
   Model:
     (model
-      (symbol_0 f32 (f32 -0.000000)))
+      (symbol_2 i32 (i32 -2147479552))
+      (symbol_0 f32 (f32 34.000000)))
   Trap: unreachable
   Model:
     (model
       (symbol_1 f64 (f64 2.000000))
+      (symbol_2 i32 (i32 8))
       (symbol_0 f32 (f32 nan)))
   Trap: unreachable
   Model:
     (model
-      (symbol_1 f64 (f64 nan))
+      (symbol_1 f64 (f64 1.000244))
       (symbol_2 i32 (i32 0))
       (symbol_0 f32 (f32 nan)))
   Trap: unreachable
   Model:
     (model
-      (symbol_1 f64 (f64 nan))
+      (symbol_1 f64 (f64 1.031494))
       (symbol_2 i32 (i32 1))
       (symbol_0 f32 (f32 nan)))
   Trap: unreachable
   Model:
     (model
-      (symbol_1 f64 (f64 606188394992561843776909558182978781184.000000))
-      (symbol_2 i32 (i32 33930))
+      (symbol_1 f64 (f64 -338460706455329128135018695364373065576360054171204343201037173805357995568625611429596369334345869072216864991017687935090688.000000))
```

Comment:
Hahaha. We should probably try to use another printing function for big floats like this. I guess OCaml has a way to print floats in scientific notation. cc @filipeom

Reply:
Hahah, yes this isn't an easy number to decode :sweat_smile: I can fix this directly in the encoding, it should be easy.

Reply:
Ooops, scientific notation is not going to help here, I meant hexadecimal.

Reply:
You don't like [this](https://github.com/formalsec/encoding/blob/main/test/test_qf_bvfp.expected) representation? The `Model.pp` is just for display purposes, the model we output in the `.xml` files come in hexadecimal so there won't be any loss in precision.

But I'm not opposed to print these numbers in hexadecimal. Do you think every IEEE float and bitvec integer should be printed in hexadecimal, or just the big ones?

Reply:
Yes I was worried about not being able to get the exact value. When is the `.xml` file generated (I don't remember sorry) ? I'm fine with the current display if there's a way to get the exact value somehow. :)

Reply:
It's generated at the end of every feasible branch [here](https://github.com/OCamlPro/owi/blob/71fce491657e5465b715debfab7f045a1b4c1e51/src/cmd_sym.ml#L307). 

In the encoding there's two functions to print values: `pp` that displays floats regularly and `pp_num` that displays fps and bitvectors in hexadecimal. But I should probably improve this and create a more explicit function for this purpose